### PR TITLE
Fixed css for cart and products

### PR DESCRIPTION
**Pull request by sahag98**

### CODE REVIEW

**Review by Noya-19 — CHANGES_REQUESTED** [review]
- [ ] Change cart colors to not blend in with the main site
- [ ] Add white background to each product

File: client/src/components/ProductCard.vue [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+        <img :src="image" style="width:100%;" />
+        <div class="product-title">
             {{ product.name }}
         </div>
+        
         <div class="card-body">
             <form>
+                <div class="row">
+                    <div class="cell">
+                        <label class ="image"><font-awesome-icon icon="fa-tshirt"></label>
```
Comment (Noya-19) [inline client/src/components/ProductCard.vue:13]: Add closing tag.
When testing, no images show up. Check to see if works

**Review by Noya-19 — CHANGES_REQUESTED** [review]

**Review by HeroofLight555 — APPROVED** [review]
Looks Good

**Review by sahag98 — COMMENTED** [review]
Fixed

**Review by sahag98 — COMMENTED** [review]
Looks good

**Pull request merged**